tests/test_more.py

```diff
@@ -5803,3 +5804,18 @@ def test_basic(self):
 
     def test_empty(self):
         self.assertEqual(dict(mi.join_mappings()), {})
+
+
+class DiscreteFourierTransformTests(TestCase):
+    def test_basic(self):
+        xarr = [1, 2 - 1j, -1j, -1 + 2j]
```

Comment:
Suggest adding a comment that links back to the wikipedia example which is worked out step-by-step: `# https://en.wikipedia.org/wiki/Discrete_Fourier_transform#Example`.